server/src/routes/comments.js

```diff
@@ -4,20 +4,36 @@ const { firebaseAuthMiddleware } = require('../middleware/firebaseAuth');
 const router = express.Router();
 const Post = require('../db/models/post');
 const Comment = require('../db/models/comments');
+const User = require('../db/models/users');
 
 // Add one comment
 // eslint-disable-next-line no-unused-vars
 router.post('/', firebaseAuthMiddleware, async (req, res) => {
-  const comment = new Comment({
-    parentID: req.body.parentID,
-    body: req.body.body,
-  });
   try {
+    let user = await User.findOne({
+      email: req.user.email,
+    });
+
+    if (user == null) {
```

Comment:
Just a side note. null is falsy which allows `if(!user)` which will also cover the case when the user is undefined (which is more common than it being set to null)